src/browser/suggestion-box.js

```diff
@@ -132,7 +132,7 @@
       pointerEvents: 'none'
     },
     collapsed: {
-      visibility: 'collapse'
+      display: 'none'
```

Comment:
This breaks screenshot feature (See https://github.com/mozilla/browser.html/blob/master/src/browser/web-view.js#L354-L357) #266